runtime/testing-basilisk/src/lib.rs

```diff
@@ -116,7 +116,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("testing-basilisk"),
 	impl_name: create_runtime_str!("testing-basilisk"),
 	authoring_version: 1,
-	spec_version: 22,
+	spec_version: 23,
```

Comment:
@enthusiastmartin i think that both runtime versions should be always in sync even if nothing really changed in other runtime

Reply:
> @enthusiastmartin i think that both runtime versions should be always in sync even if nothing really changed in other runtime

Ok. I updated the runtime version to match the testing runtime version. Although , I am not convinced with the testing runtime done this way ( but that's another discussion )